tensorboard/loader_test.py

```diff
@@ -221,7 +221,10 @@ def testNoDataSinceLastUpdate_displaysStalledAlert(self):
     self.assertIn('[stalled]', self.logs[1])
 
   def testBigNumberInAmerica_showsCommas(self):
-    locale.setlocale(locale.LC_ALL, 'en_US.utf8')
+    try:
+      locale.setlocale(locale.LC_ALL, 'en_US.utf8')
```

Comment:
Hmm, perhaps we can add a second fallback for the `en_US.UTF-8` variant which I think is just an alias?  E.g.

```python
try:
  locale.setlocale(locale.LC_ALL, 'en_US.utf8')
except Exception:
  try:
    locale.setlocale(locale.LC_ALL, 'en_US.UTF-8')
  except Exception:
    raise self.skipTest('Environment does not support en_US.utf8/en_US.UTF-8 locale')
```

Reply:
Sure let me do it

Reply:
Updated

Reply:
I checked our internal codebase and `en_US.UTF-8` is cited 10x more times than `en_US.utf8`. Maybe we could get rid of the try/except and simply say `en_US.UTF-8`.

Reply:
Hmm, yeah, it sounds like `utf8` is really the variant and `UTF-8` is the standard one according to this: https://superuser.com/questions/999133/differences-between-en-us-utf8-and-en-us-utf-8

Indeed on my machine they both seem to normalize to `UTF-8` when actually retrieved via `getlocale()`:
```python
>>> import locale
>>> locale.getdefaultlocale()
('en_US', 'UTF-8')
>>> locale.getlocale()
(None, None)
>>> locale.setlocale(locale.LC_ALL, 'en_US.utf8')
'en_US.utf8'
>>> locale.getlocale()
('en_US', 'UTF-8')
>>> locale.setlocale(locale.LC_ALL, 'en_US.UTF-8')
'en_US.UTF-8'
>>> locale.getlocale()
('en_US', 'UTF-8')
>>> 
```

Reply:
I saw the post too, my concern is if `en_US.UTF-8` is cited more than `en_US.utf8` in your codebase it means some ppl are still using `en_US.utf8` on their systems right?
But, yeah, if `UTF-8` is more standard I agree we can stick to it, let me do the update, thanks for your replies!

Reply:
Sometimes certain systems environments are too lenient, which leads us to make mistakes like these. It's unfortunate. I'd prefer the canonical representation.

Reply:
Somehow Travis CI is failing 😞